packages/babel-plugin-ember-test-metadata/README.md

```diff
@@ -111,4 +111,37 @@ module.exports = function (defaults) {
 
   return app.toTree();
 };
+```
+
+You can also pass a custom function to normalize your test file path.
+```js
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function (defaults) {
+  function customNormallizedFilePath(filePath, packageName, projectRoot) {
```

Comment:
Typo in `customNormallizedFilePath`.
customNormallizedFilePath -> customNormalizedFilePath